tests/Modules/Okapi/Integration/Services/ApiSrvTest.php

```diff
@@ -34,6 +34,6 @@ public function testApiSrvInstallationsMethod()
 
         $response = $client->get('services/apisrv/installations');
 
-        self::assertGreaterThanOrEqual(7, $response);
+        self::assertGreaterThanOrEqual(5, $response);
```

Comment:
installations may be shut down or removed for other reasons

Reply:
if this happen I want to realize it.